public/paad_cptac_2021/meta_study.txt

```diff
@@ -0,0 +1,9 @@
+add_global_case_list: true
+type_of_cancer: paad
+name: Pancreatic ductal adenocarcinoma. (CPTAC, Cell 2021)
+description: Proteogenomic characterization of pancreatic ductal adenocarcinoma. Generated by CPTAC.
+short_name: Proteogenome of human PDAC (CPTAC Cell2021)
```

Comment:
I left `short_name` as it would otherwise fail the validator.
I also kept the PDAC abbreviation as the cancer abbreviation in the paper title, and paad for the cancer named used in cbioportal.